### DISCUSSION TITLE
Whitebeet User Manual V2.14 - Set DC Charging Parameters (18.9.3)

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#124] [General]
Can you confirm that the example in the manual for 'Set DC Charging Parameters' is correct? The Departure Time is defined as being 'uint32', but only 2 bytes are used in the example?

Cheers

Rob

**Comment by jpo-stx** [comment]
I can confirm that the departure time parameter is of type uint32.

The example needs to be fixed.
